test/src/planning/VisitorTests.cc

```diff
@@ -127,4 +129,23 @@ TEST_F(VisitorTestFixture, testPartitionSubqueryUnion) {
     auto encoded = transcoded.Encode(Codec::hevc());
 
     Coordinator().execute(encoded);
-}
\ No newline at end of file
+}
+
+TEST_F(VisitorTestFixture, testScanInterpolateDiscretize) {
+    auto outputResolution = 416;
+
+    auto input = Scan(Resources.red10.name);
+    auto continuous = input.Interpolate(Dimension::Theta, interpolation::Linear());
+    auto smallTheta = continuous.Discretize(Dimension::Theta, rational_times_real({2, outputResolution}, PI));
```

Comment:
I've been meaning to include a `radians` and `degrees` wrapper so these verbose constructs aren't necessary...